common/buildcraft/factory/tile/ITileLed.java

```diff
@@ -0,0 +1,8 @@
+package buildcraft.factory.tile;
+
+public interface ITileLed {
+    boolean isDone();
+    int getPowerLevel();
+    int getX();
+    int getY();
```

Comment:
These aren't very clear - how about ````getLedX```` and ````getLedY```` as then its obvious that it doesn't refer to the co-ords of the tile.